man/mono.1

```diff
@@ -1652,11 +1652,11 @@ Automatically generates sequence points where the
 IL stack is empty.  These are places where the debugger can set a
 breakpoint.
 .TP
-\fBgen-compact-seq-points\fR
```

Comment:
It's used in monodroid at least: https://github.com/xamarin/xamarin-android/blob/14260f43fc47abc9aee0ff5a0ca10a10ac2e3384/src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs#L324 (and who knows what other code relies on it).

We should just make this a no-op now that it's the default.